src/test/test-notify-recv.c

```diff
@@ -80,7 +98,16 @@ TEST(notify_socket_prepare) {
         if (r == 0) {
                 ASSERT_OK_ERRNO(setenv("NOTIFY_SOCKET", path, /* overwrite = */ true));
                 ASSERT_OK_POSITIVE(sd_notify(/* unset_environment = */ false, "FIRST_MESSAGE=1"));
-                ASSERT_OK_POSITIVE(sd_notify(/* unset_environment = */ false, "FIRST_MESSAGE=2\nADDITIONAL_DATA=hoge"));
+
+                _cleanup_close_ int fd1 = open("/tmp", O_RDONLY|O_CLOEXEC|O_DIRECTORY);
+                ASSERT_OK_ERRNO(fd1);
+                _cleanup_close_ int fd2 = open("/dev/null", O_RDONLY|O_CLOEXEC);
+                ASSERT_OK_ERRNO(fd2);
+
+                ASSERT_OK_POSITIVE(
+                        sd_pid_notify_with_fds(
+                                0, /* unset_environment = */ false,
+                                "FIRST_MESSAGE=2\nADDITIONAL_DATA=hoge", (int[]) { fd1, fd2 }, 2));
```

Comment:
Should this be `"SECOND_MESSAGE=1\nADDITIONAL_DATA=hoge"` to match the check on line 52? (This was a pre-existing mismatch.)

Reply:
-> #37034.